src/nvim/drawline.c

```diff
@@ -1135,10 +1090,9 @@ int win_line(win_T *wp, linenr_T lnum, int startrow, int endrow, bool number_onl
     }
 
     // Check for columns to display for 'colorcolumn'.
-    color_cols = wp->w_buffer->terminal ? NULL : wp->w_p_cc_cols;
-    if (color_cols != NULL) {
-      draw_color_col = advance_color_col(VCOL_HLC, &color_cols);
-    }
+    wlv.color_cols = wp->w_buffer->terminal ? NULL : wp->w_p_cc_cols;
+    wlv.draw_color_col = wlv.color_cols;
```

Comment:
Without the `!= NULL` it looks a bit strange
```suggestion
    wlv.draw_color_col = wlv.color_cols != NULL;
```